cmd/machine-config-daemon/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - ashcrow
```

Comment:
Can we get @jlebon added here as well? He also will be highly involved in this.

Reply:
:+1: done